common/test/acceptance/pages/studio/import_export.py

```diff
@@ -10,6 +10,19 @@
 from . import BASE_URL
 
 
+class TemplateCheckMixin(object):
```

Comment:
This appears to only be called by the tests.

Seems like a lot of work to generate a property that only appears to be used within tests. Might be better for the test to monkey-patch this in when needed.

My concern is that this will totally work, but might be really difficult to maintain - consider someone else coming back in 6 months to look at this!